lips/lip-6.md

```diff
@@ -1,7 +1,7 @@
 ---
 lip: 6
 title: Libra Validator Configuration Management
-authors: Valeria Nikolaenko, David Wolinsky, Sam Blackshear, David Dill
+author: Valeria Nikolaenko, David Wolinsky, Sam Blackshear, David Dill
```

Comment:
This has to stay `authors` when there are multiple authors as that helps with the parsing via the `,` separator.